manim/mobject/mobject.py

```diff
@@ -2538,9 +2528,30 @@ def add_n_more_submobjects(self, n):
     def repeat_submobject(self, submob):
         return submob.copy()
 
-    def interpolate(self, mobject1, mobject2, alpha, path_func=straight_path()):
-        """Turns this :class:`~.Mobject` into an interpolation between ``mobject1``
-        and ``mobject2``.
+    def interpolate(
+        self,
+        mobject1: "Mobject",
+        mobject2: "Mobject",
+        alpha: float,
+        path_func: Optional[Callable] = None,
+    ) -> "Mobject":
+        """Turns this :class:`~.Mobject` into an interpolation between ``mobject1`` and ``mobject2``.
```

Comment:
I don't think this docstring is helpful.

Reply:
Despite what git claims, I did not write that. It's been there.